rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/TagsModule.scala

```diff
@@ -46,3 +49,5 @@ trait TagsModule {
     }
   }
 }
+
+object TagsModule extends TagsModule
```

Comment:
let's just use `new TagsModule {}` in tests, since this object isn't used anywhere in production code